ethportal-peertest/src/jsonrpc.rs

```diff
@@ -22,13 +22,11 @@ use trin_core::{
 const DATA_RADIUS: Distance = Distance::MAX;
 /// Default enr seq value
 const ENR_SEQ: &str = "1";
-/// History header content key & value for merge block on mainnet
-/// Use merge block here so that we can use the default master accumulator for validation
-/// rather than needing to go lookup an epoch accumulator.
+/// History HeaderWithProof content key & value
 pub const HISTORY_CONTENT_KEY: &str =
-    "0x0055b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286";
+    "0x006251d65b8a8668efabe2f89c96a5b6332d83b3bbe585089ea6b2ab9b6754f5e9";
 pub const HISTORY_CONTENT_VALUE: &str =
```

Comment:
The reason we used the merge block before, was due to our structure of the `MasterAccumulator`, we could verify the merge block directly without needing to rely on an `EpochAccumulator`. Actually, I believe the choice for this block was even before we had support for validation via an accumulator proof. Basically, our validation required the `EpochAccumulator` in which it looked up the block hash directly and made sure they matched. Now, all headers have proofs, and we no longer require an `EpochAccumulator` for validation. This value is just one of the test vectors from `trin-core/src/assets/test/fluffy/header_with_proofs.json` since I didn't feel there was a need to generate a `HeaderWithProof` content value for the merge block. I added the block number for reference